src/components/addEvent/AddEvent.js

```diff
@@ -1,10 +1,96 @@
 import './AddEvent.css';
+import { useState } from 'react';
+import { useMutation, gql } from "@apollo/client";
 
-export const AddEvent = () => {
+const ADD_NEW_EVENT = gql`
+  mutation ($name: String!, $dogId: Int!, $eventDatetime: ISO8601DateTime!){
+    createEvent(input: {
+      name: $name,
+      dogId: $dogId,
+      completed: false,
+      eventDatetime: $eventDatetime,  
+  }) {
+      event {
+        id
+        dogId
+        name
+        completed
+        eventDatetime
+      }
+      errors
+    }
+  }
+  `;
+
+export const AddEvent = ({dogs}) => {
+  const [eventName, setEventName] = useState('');
+  const [eventDate, setEventDate] = useState('');
+  const [dogId, setDogId] = useState(0)
+  const [error, setError] = useState('');
+  const [addNewEvent] = useMutation(ADD_NEW_EVENT)
+
+  // {
+  //   refetchQueries: []
```

Comment:
I think there is a way to update our events without performing another query. We have discussed this and scheduled a time to go over implementation. Currently, the plan is to make another PR dedicated to this implementation.